<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SendgroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>发送数据</string>
    </property>
    <widget class="QLabel" name="SendHead">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帧 头：</string>
     </property>
    </widget>
    <widget class="QLabel" name="SendData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数 据：</string>
     </property>
    </widget>
    <widget class="QLabel" name="SendLast">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>校验码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帧  尾：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SendHeadLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="SendDatatextEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="SendCRCLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="SendLastLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="HexSend">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制(空格分隔)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AutoSend">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>定时发送</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="AutoSendTime">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendDataSUM">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>求SUM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendDataCRC16">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>求CRC16</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendDataPush">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendDataClear">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ReceivegroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>631</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>接收数据</string>
    </property>
    <widget class="QTextBrowser" name="ReceiveDatatextBrowser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>461</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="HexReceive">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReceiveSave">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>另存为</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReceiveStop">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ReceiveClear">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ComStatuslabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口已关闭！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Fail">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="Save"/>
    <addaction name="Quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>通讯</string>
    </property>
    <addaction name="Communating"/>
    <addaction name="DisCommunated"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="About"/>
   </widget>
   <addaction name="Fail"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Save"/>
   <addaction name="Communating"/>
   <addaction name="DisCommunated"/>
   <addaction name="About"/>
   <addaction name="Quit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Save">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="Quit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="Communating">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="DisCommunated">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </action>
  <action name="About">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
